examples/operator/values.yaml

```diff
@@ -44,6 +44,8 @@ controllerManager:
     region: east
     type: user-node
   replicas: 1
+  serviceaccount:
```

Comment:
@hfuss where does this come from? The generated serviceaccount.yaml says `{{ toYaml .Values.controllerManager.annotations | nindent 4 }}` and I don't see any changes to my `values.yaml` file.

Reply:
@arttor @hfuss I believe this this is the correct template https://github.com/arttor/helmify/commit/e7f15294fbaee735362198d050693340e5dd8a61

Reply:
I'll do that @arttor 😄

Reply:
Thanks @Starefossen for picking up my slack and seeing this through! 🚀